File: src/main/java/com/dv/telegram/google/RowData.java
package com.dv.telegram.google;

import lombok.Data;

import java.util.ArrayList;
import java.util.List;

@Data
public class RowData {

    private List<String> cells;

    public RowData() {
        this.cells = new ArrayList<>();
    }

    public void addCell(String cell) {
        cells.add(cell);
    }

    public String getCellOrBlank(int index) {
        if ((0 <= index) && (index < cells.size())) {
            return cells.get(index);
        }

        return "";
    }
}

File: src/main/kotlin/com/dv/telegram/google/RowData.kt
package com.dv.telegram.google

class RowData {
    val cells: MutableList<String> = mutableListOf()

    fun addCell(cell: String) = cells.add(cell)

    fun getCellOrBlank(index: Int): String = cells.getOrElse(index) { "" }
}
